Composite/Composite.Cs.Tests/Composite/ToComponentsTests.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Xunit;
+
+using Composite;
+
+namespace Composite.Cs.Tests
+{
+    public class ToComponentsTests
+    {
+        [Fact]
+        public void TransformationTest(){
```

Comment:
The test does not check a transformation logic, so it can be renamed to LazyInputTest.